### PR TITLE
Fix overeager metadata loading in datamodel

### LINKED ISSUE #3276: Data Model page makes 60+ API calls, 40 of which are duplicates; crashing servers due to overload
*  I am using Chrome 52
*  My computer's OS is Mac OS X 10.11.4
*  I'm running Metabase on Heroku and it is using the Hobby Basic Postgres DB.
*  My data is in a Postgres database
*  My Metabase version is 0.19.3 (latest per the app)

When I click on `Data Model` in `Site Administration` in an attempt to create segments, I get a blank page:

image

I've setup a few dashboards, a handful of cards, integrated SSO, added a Pulse, and upgraded my PG add-on. I'm also running as a 2X dyno in Heroku. This appears to be the only area not functioning.

Would love some insight on how to get past this issue.

### PR DESCRIPTION
**Pull request by tlrobinson**
Resolves #3276

### CODE REVIEW

File: frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default class PartialQueryBuilder extends Component {
                 }
             }
         };
-        let previewUrl = "/q/" + serializeCardForUrl(previewCard);
+        let previewUrl = "/q#" + serializeCardForUrl(previewCard);
```
Comment (salsakran) [inline frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx]: Stupid question but -- How's this work?

File: frontend/src/metabase/admin/datamodel/datamodel.js [hunk 2]
```diff
@@ -19,15 +18,8 @@ const Metabase = new AngularResourceProxy("Metabase", ["dataset"]);
...
-async function loadDatabaseMetadata(databaseId) {
-    let databaseMetadata = await MetabaseApi.db_metadata({ 'dbId': databaseId });
-
-    databaseMetadata.tables = await Promise.all(databaseMetadata.tables.map(async (table) => {
-        table = await augmentTable(table);
-        return table;
-    }));
-
-    return databaseMetadata;
+function loadDatabaseMetadata(databaseId) {
```
Comment (salsakran) [inline frontend/src/metabase/admin/datamodel/datamodel.js]: is the idea here that db_metadata contains ll the table metadata already?

**Review by salsakran — COMMENTED** [review]
think it's a +1 but I don't understand the `#` vs `/` distinction

File: frontend/src/metabase/admin/datamodel/datamodel.js (hunk 2 above)
Comment (tlrobinson) [inline frontend/src/metabase/admin/datamodel/datamodel.js]: Yes, not exactly sure why we were loading it separately as well.

File: frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx (hunk 1 above)
Comment (tlrobinson) [inline frontend/src/metabase/admin/datamodel/components/PartialQueryBuilder.jsx]: We switched from /q/... to /q#... awhile ago (#1758). There used to be a redirect for the legacy URLs so this wasn't totally broken, but I think it was removed at some point in the transition from Angular, so it was broken. This is the correct fix.

**Pull request merged**